library/java/org/chromium/net/FakeX509Util.java

```diff
@@ -20,6 +24,10 @@ public static void addTestRootCertificate(byte[] rootCertBytes) {
 
   public static void clearTestRootCertificates() { validFakeCerts.clear(); }
 
+  /* Fake certificate chain verification. Returns CertVerifyStatusAndroid.OK only if all
```

Comment:
nit: It would be nice to make the first part of this comment a complete sentence. Perhaps something like "Performs fake certificate chain verification."? Also, should the comment start with /**?

Reply:
Done, thanks!